core/framework/Settings.php

```diff
@@ -148,7 +148,7 @@
 
         protected static $_ver_mj = 4;
         protected static $_ver_mn = 1;
-        protected static $_ver_rev = 10;
+        protected static $_ver_rev = 9;
```

Comment:
Does this actually need a settings version number bump? Unless there are database changes in here, this shouldn't be necessary. Could you please rebase this off the current master?